foursight_core/react/api/jwt_utils.py

```diff
@@ -61,6 +61,7 @@ def jwt_decode(jwt: str, audience: str, secret: str) -> dict:
         # This decoding WITH signature verification is very fast;
         # on my (dmichaels) MacBook it generally takes less than 0.04ms;
         # very good since we do this on every (protected) React API call.
-        return jwtlib.decode(jwt, secret, audience=audience, leeway=30, options={"verify_signature": True}, algorithms=["HS256"])
+        args = {"leeway": 30, "options": {"verify_signature": True}, "algorithms": ["HS256"]}
```

Comment:
Yes, see the **args in the call. But maybe you're referring to the fact that the variable name in this case should really be ``kwargs`` rather than ``args``.